examples/webgl_raycast_sprite.html

```diff
@@ -0,0 +1,163 @@
+        <style>
+            body {
+                margin: 0px;
+                background-color: #fff;
+                overflow: hidden;
+            }
+
+            #info{
+                position: absolute;
+                z-index: 1;
+                width: 100%;
+                text-align: center;
+            }
+        </style>
+    </head>
+<body>
+    <div id="info"><a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> - webgl raycast sprite<br></div>
+    <script src="../build/three.js"></script>
+    <script src="./js/controls/OrbitControls.js"></script>
+    <script>
+        var renderer;
+        var scene;
+        var spriteGroup;
+        var camera;
+        var controls;
+
+        init();
+
+        function init() {
+
+            // init renderer
+            renderer = new THREE.WebGLRenderer( {
+                antialias: true
+            } );
+            renderer.setSize( window.innerWidth, window.innerHeight );
+            document.body.appendChild( renderer.domElement );
+            renderer.setClearColor( 0xFFFFFF, 1.0 );
+
+            // init scene
+            scene = new THREE.Scene();
+            spriteGroup = new THREE.Object3D();
+            scene.add( spriteGroup );
+
+            // init camera
+            camera = new THREE.PerspectiveCamera( 50, window.innerWidth / window.innerHeight, 1, 1000 );
+            camera.position.set( 15, 15, 15 );
+            camera.lookAt( scene.position );
+
+            controls = new THREE.OrbitControls( camera, renderer.domElement );
+            controls.enableRotate = true;
+
+            // add sprites
+            var sprite;
+
+            sprite = makeSprite( '#69f', spriteGroup );
+            sprite.position.set( 6, 5, 5 );
+            sprite.scale.set( 2, 5, 1 );
+
+            sprite = makeSprite( '#69f', spriteGroup );
+            sprite.position.set( 8, - 2, 2 );
+            sprite.center.set( 0.5, 0 );
+            sprite.scale.set( 1, - 5, 1 );
+            sprite.material.rotation = Math.PI / 3 * 4;
+
+            var g = new THREE.Object3D();
+            spriteGroup.add( g );
+            g.scale.set( 1, 2, 1 );
+            g.position.set( - 5, 0, 0 );
+            g.rotation.set( Math.PI / 2, 0, 0 );
+
+            sprite = makeSprite( '#69f', g );
+            sprite.position.set( 0, 2, 5 );
+            sprite.scale.set( 10, 2, 3 );
+            sprite.center.set( - 0.1, 0 );
+            sprite.material.rotation = Math.PI / 3;
+
+            window.addEventListener( 'resize', onWindowResize, false );
+            window.addEventListener( "mousemove", onDocumentMouseMove, false );
+
+            render();
+        }
+
+        function makeSprite( color, parent ) {
+
+            var sprite = new THREE.Sprite( new THREE.SpriteMaterial( {
+                color: color
+            } ) );
+            parent.add( sprite );
+            return sprite;
+
+        }
+
+        function render() {
```

Comment:
Please rename `render()` to `animate()` and call it right after `init()` in line 34 so you have this workflow.

```js
init();
animate();
```